src/vm/argdestination.h

```diff
@@ -42,31 +45,17 @@ class ArgDestination
         return dac_cast<PTR_VOID>(dac_cast<TADDR>(m_base) + m_offset);
     }
 
-#if defined(UNIX_AMD64_ABI) && defined(FEATURE_UNIX_AMD64_STRUCT_PASSING)
+#if (defined(UNIX_AMD64_ABI) && defined(FEATURE_UNIX_AMD64_STRUCT_PASSING)) || defined(_TARGET_ARM64_)
 
     // Returns true if the ArgDestination represents a struct passed in registers.
     bool IsStructPassedInRegs()
     {
         LIMITED_METHOD_CONTRACT;
+#if defined(_TARGET_ARM64_)
+        return m_argLocDescForStructInRegs != NULL;
```

Comment:
I'm not sure I understand your comment. I've just shown that we are not setting m_argLocDescForStructInRegs for all arguments.